assets/src/design-system/components/checkbox/index.js

```diff
@@ -37,7 +37,6 @@ const Border = styled.div(
     width: ${CONTAINER_WIDTH}px;
     border-radius: ${theme.borders.radius.small};
     border: ${BORDER_WIDTH}px solid ${theme.colors.border.defaultNormal};
-    pointer-events: none;
```

Comment:
I've seen `pointer-events: none` as a fix to make inner elements clickable when they are wrappedby elements that don't typically have onChange or onClick events. I'm sure you tested this in storybook so it probably won't be an issue. Didn't look like the DS checkbox is used yet though in dashboard/edit-story so I just wanted to flag it. :)

Reply:
i was wary of removing this, but it's working just fine and all tests are passing! The checkbox is used on the telemetry banner as well as the dashboard settings page. I tested chrome, firefox, safari and everything is as it should be. I'll keep an eye out for anything being amiss here, but for now i think all is well! Thanks for flagging it!